@here/harp-materials/lib/Utils.ts

```diff
@@ -77,15 +98,36 @@ export function enableBlending(
         material.blendSrcAlpha = THREE.OneFactor;
         material.blendDstAlpha = THREE.OneMinusSrcAlphaFactor;
     }
+    if (isUpdate && material instanceof THREE.Material) {
+        material.needsUpdate = true;
+    }
 }
 
+/**
+ * Disable alpha blending using THREE.CustomBlending mode, switches to [[THREE.NormalBlending]].
+ *
+ * @see enableBlending.
+ * @param material The material or material parameters to modify.
+ * @param isUpdate Set to true if you want to notify about material change, required only if you
+ * change blend mode after material has been created (in frame update), it does not have meaning
+ * when you pass [[THREE.ShaderMaterialParameters]] into function. By default set to [[false]].
+ */
 export function disableBlending(
-    material: (THREE.Material | THREE.ShaderMaterialParameters) & ForcedBlending
+    material: (THREE.Material | THREE.ShaderMaterialParameters) & ForcedBlending,
+    isUpdate: boolean = false
 ) {
-    if (material.transparent || material.forcedBlending) {
+    if (
+        material.transparent ||
+        material.forcedBlending ||
+        material.blending === THREE.NormalBlending
```

Comment:
Second most important part of the fix - do not force material update when blending mode stays the same.